thirdparty/CMakeLists.txt

```diff
@@ -151,11 +151,3 @@ install(DIRECTORY imgui/misc DESTINATION ${MESHLIB_THIRDPARTY_INCLUDE_DIR}/imgui
 install(DIRECTORY eigen/Eigen DESTINATION ${MESHLIB_THIRDPARTY_INCLUDE_DIR})
 install(DIRECTORY eigen/unsupported DESTINATION ${MESHLIB_THIRDPARTY_INCLUDE_DIR})
 
-if(APPLE)
-  set(BUILD_SHARED_LIBS ON)
```

Comment:
this looks common

Reply:
@Grantim this part is revert of https://github.com/MeshInspector/MeshLib/pull/2311

@oitel what do you think?

Reply:
@Grantim This option is already setted above. So I think nothing should be changed.